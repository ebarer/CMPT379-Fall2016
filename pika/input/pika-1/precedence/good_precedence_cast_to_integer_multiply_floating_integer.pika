exec {

  print [ 9.4 | int ] * 12, _n_.
  print [ 4.5 | int ] * 2, _n_.
  print [ 4.7 | int ] * 20, _n_.
  print [ 8.5 | int ] * 41, _n_.
  print [ 8.9 | int ] * 73, _n_.
  print [ 7.9 | int ] * 20, _n_.
  print [ 4.5 | int ] * 64, _n_.
  print [ 8.8 | int ] * 19, _n_.
  print [ 2.7 | int ] * 79, _n_.
  print [ 8.6 | int ] * 72, _n_.
  print [ 4.8 | int ] * 32, _n_.
  print [ 1.4 | int ] * 5, _n_.

  print 65 * [ 6.1 | int ], _n_.
  print 67 * [ 4.8 | int ], _n_.
  print 15 * [ 5.1 | int ], _n_.
  print 47 * [ 2.2 | int ], _n_.
  print 20 * [ 7.3 | int ], _n_.
  print 94 * [ 5.5 | int ], _n_.
  print 83 * [ 2.2 | int ], _n_.
  print 14 * [ 7.6 | int ], _n_.
  print 25 * [ 8.2 | int ], _n_.
  print 30 * [ 5.3 | int ], _n_.
  print 19 * [ 8.3 | int ], _n_.
  print 48 * [ 2.3 | int ], _n_.

  print [ 6.5 | int ] * [ 1.3 | int ], _n_.
  print [ 7.6 | int ] * [ 6.6 | int ], _n_.
  print [ 6.3 | int ] * [ 7.2 | int ], _n_.
  print [ 4.3 | int ] * [ 2.2 | int ], _n_.
  print [ 3.4 | int ] * [ 6.9 | int ], _n_.
  print [ 6.0 | int ] * [ 6.3 | int ], _n_.
  print [ 6.7 | int ] * [ 4.2 | int ], _n_.
  print [ 9.9 | int ] * [ 4.5 | int ], _n_.
  print [ 8.4 | int ] * [ 5.9 | int ], _n_.
  print [ 2.3 | int ] * [ 5.6 | int ], _n_.
  print [ 7.9 | int ] * [ 8.2 | int ], _n_.
  print [ 3.7 | int ] * [ 6.1 | int ], _n_.

}

