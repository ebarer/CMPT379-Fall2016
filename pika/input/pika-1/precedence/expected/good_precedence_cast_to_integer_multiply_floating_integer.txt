108
8
80
328
584
140
256
152
158
576
128
5
390
268
75
94
140
470
166
98
200
150
152
96
6
42
42
8
18
36
24
36
40
10
56
18
